src/objc/rfc822/MCOMessageParser.mm

```diff
@@ -16,6 +16,10 @@
 #import "MCOUtils.h"
 #import "MCOAbstractMessageRendererCallback.h"
 
+@interface MCOMessageParser ()
```

Comment:
Necessary because the initializer is declared in a private extension and cannot be marked designated